install/ossm-cert-manager.adoc

```diff
@@ -10,37 +10,27 @@ toc::[]
 
 The cert-manager tool is a solution for X.509 certificate management on Kubernetes. It delivers a unified API to integrate applications with private or public key infrastructure (PKI), such as Vault, Google Cloud Certificate Authority Service, Let's Encrypt, and other providers.
 
+The cert-manager tool ensures the certificates are valid and up-to-date by attempting to renew certificates at a configured time before they expire.
```

Comment:
```suggestion
The cert-manager tool ensures that the certificates are valid and up-to-date by attempting to renew certificates at a configured time before they expire.
```
Is it the "cert-manager tool" or the "cert-manager Operator"?

Reply:
This is an explanation about the generic cert-manager tool which is a cloud native certificate management tool. Cert-manager Operator is Red Hat Openshift specific which is a cluster-wide service that provides application certificate lifecycle management.